### DISCUSSION TITLE
mesh surface for the DICOM image

### DISCUSSION THREAD

**Discussion by zuiozz** [mlavik1/UnityVolumeRendering#169] [Ideas]
I was just thinking if I can add a mesh surface on the volume using the isosurface rendering or maybe other rendering methods so I can get a model of what I visualised with certain threshold in order to cut the volume of interest of and calculate volume of that region. But I have no idea where to start, can you help me with this?

**Comment by mlavik1** [comment]
Hi @zuiozz !
This is not something the plugin offers, but I'd recommend you to have a look at the [marching cubes](https://en.wikipedia.org/wiki/Marching_cubes) or similar algorithms. 
A quick search on GitHub shows that there are some implementation of it there.
However, maybe a simpler tool could be to use soom tool to export a mesh from the DICOM, and then import the mes into Unity?
I have no experience with it myself, but there appears to be quite a few tools that can do this:
- [3D Slicer](https://www.youtube.com/watch?v=ASXNYydw76s)
- [DicomToMesh](https://github.com/AOT-AG/DicomToMesh)

It seems like [ImageVis3D](https://www.sci.utah.edu/software/imagevis3d.html) also has this features, as mentioned here: http://www.sci.utah.edu/~tfogal/iv3d/manual/ImageVis3D.pdf
